### DISCUSSION TITLE
Support for Spotify free

### DISCUSSION THREAD

**Discussion by javilopezalarcon** [music-assistant/support#612] [Feature requests and ideas]
good morning! I love the line that this integration is taking, thanks! I was thinking if it would be possible to integrate spotify but for those of us who have a free subscription, for example, spotcast works (https://github.com/fondberg/spotcast), which works in free versions. and playlists can be launched either by putting the url or with the spotify lovelance card (https://github.com/custom-cards/spotify-card) it would be fine for users who cannot pay the subscription.

**Comment by marcelveldt** [comment]
Support for free Spotify accounts is most probably never going to happen because of the simple fact that MA is a queue streamer and it can't stream the Spotify commercials belonging to a free account unless that is supported by librespot some day ( I think it won't).

So I'm going to close this as feature request due to the simple fact that it is technically impossible.
